### PR TITLE
Helm: Add distributed URLs to nginx gateway

### PR DESCRIPTION
**Pull request by DylanGuedes**
**What this PR does / why we need it**:
Modify our nginx proxy pass to redirect to microservices components correctly.
Modify index-gateway to run in simple mode by default.
Fix distributor svc target port (http-metrics instead of `http`).

**Which issue(s) this PR fixes**:
N/A

### CODE REVIEW

**Comment by JStickler** [comment]
Gentle reminder, from the PR checklist "For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`."

**Comment by DylanGuedes** [comment]
> Gentle reminder, from the PR checklist "For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`."

This is part of a secondary helm chart initiative that isn't released yet. The target branch is `distributed-helm-chart` instead of `main`.

**Review by MichelHollands — APPROVED** [review]
LGTM

File: production/helm/loki/values.yaml [hunk 1]
```diff
@@ -362,9 +362,10 @@ loki:
   ingester: {}
   # --  Optional index gateway configuration
   index_gateway:
-    mode: ring
```
Comment (lindeskar) [inline production/helm/loki/values.yaml:365]: Hi,
I'm curious to the reason behind this change. Is ring mode not recommended for Index Gateway? Please help me understand. 🙂

**Pull request merged**